Header header

float64 SystemTime
uint16  GpsWeekNumber
uint8   NumberOfSVs
uint8   ProcessingMode
float64 GpsTimeWeek
float64[3] PositionRMS
float64[3] VelocityRMS
float64[3] AttitudeRMS
